students/web-recuperada.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mi primera página web</title>
+    <h1>Titulo de nivel 1</h1>
+
+</head>
+<body>
+    <h2>Titulo de nivel 2</h2>
+    <img src="gatitos.jpg"/>
```

Comment:
missing image. You should put a valid URL o include your image as a file in your project